samples/create_table_external_data_configuration.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+
+def create_table_external_data_configuration(
+    table_id: str,
+    source_uris: List[str],
+    external_source_format: str,
+) -> None:
+    """Create a table using an external data source"""
+
+    # [START bigquery_create_table_external_data_configuration]
+    # [START bigquery_create_external_table_definition]
+    from google.cloud import bigquery
+
+    # Construct a BigQuery client object.
+    client = bigquery.Client()
+
+    """
+    TODO(developer):
+    Set table_id to the ID of the table to create.
+    table_id = "your-project.your_dataset.your_table_name"
```

Comment:
I've been using a different pattern for this. I got the occasional complaint from folks who didn't like the fact that necessary code was included in comments.

Now I reset variables like Mario does here: https://github.com/googleapis/python-bigquery/pull/1436/files#diff-933d0a59ab13b3ed5238456246f4e3fd3566082b4c2d0733949f7f885c358df8R29

Reply:
I agree with @tswast on this one.

As I have grown more familiar with our code samples and based on my experiences with students...
Having code in comments is harder to parse.
Folks tend to glaze over the comments entirely / not read them and just enter the code portions and then get perplexed when the code doesn't work at all. 

Feeding in "working code" using placeholders for things the folks need to replace at least causes the system to respond with an error code that I feel guides them more clearly to what they need to replace.